src/Elcodi/Store/PaymentBridgeBundle/Services/PaymentBridge.php

```diff
@@ -172,9 +172,13 @@ public function getExtraData()
                     ->getPurchasable()
                     ->getName();
 
+                /*
+                 * Elcodi stores amounts in integer cents (10USD = 1000)
+                 * but Paypal expects decimal (10.00USD)
+                 */
                 $orderLineArray['amount'] = $orderLine
                     ->getAmount()
-                    ->getAmount();
+                    ->getAmount() / 100;
```

Comment:
Hmm... no like. This is no Paypal related but abstract, so this stuff should be in the Paypal bundle, assuming that the bridge always must return the value without cents, right?